templates/nextflow-launch-iam-policy.yaml

```diff
@@ -0,0 +1,42 @@
+Description: Policy for NextFlow Forge service user to launch workflows
```

Comment:
The filename and PR title refer to Tower Launch, but this description refers to Tower Forge. Is "launch" referring to something else in this context?

Reply:
I see that the Tower Forge [policy](https://github.com/seqeralabs/nf-tower-aws/blob/master/forge/forge-policy.json) includes the Tower Launch policy. You can ignore this comment thread.